datapackage_pipelines_assembler/generator.py

```diff
@@ -92,10 +92,24 @@ def pipeline_id(r=None):
         urls = []
         inner_pipeline_ids = []
 
+        outputs = source.get('outputs', [])
+        zip_output = {
+            'kind': 'zip',
+            'parameters': {
+                'out-file': '%s.zip' % meta.get('dataset', 'datahub')
+            }
+        }
+        zip_there = False
```

Comment:
might be easier to understand as 

`zip_there = any(output['kind'] == 'zip' for output in outputs)`